k0776243/quiz1.py

```diff
@@ -0,0 +1,28 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+total = 0
+
+# declare an array named numbers with all numbers 0 - 20
+numbers =[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
+
+# write a for loop that processes each number
+def even(numbers):
```

Comment:
When breaking problems down it is often helpful to separate the question being asked from the action to take.

So when we say add to total if the number is even, the question being asked is is the number even and the  action to take is add to total, so I'd recommend splitting those things up. Like have your even function only tell me if a number is even or not.

It will also help to work on getting even to work on just a single number instead of the whole list because once you have it working for one it's easy to make it work for all, but making it work for one is simpler.